tensorflow/workspace.bzl

```diff
@@ -319,11 +319,13 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
       actual = "@nanopb_git//:nanopb",
   )
 
+  # zlib permalinks including the current version are always in http://zlib.net/fossils/ 
```

Comment:
I think it's it's sufficient to have this comment in the git commit log, rather than in the file -- what do you think?  Otherwise, I'll test and merge :) . Thanks!